### ISSUE TITLE
Add rotating block-specific tips to the inserter's help panel

### ISSUE THREAD

**Issue by kjellr** [WordPress/gutenberg#17091]
We should add block-specific tips in the inserter panel that was added in #16813. These could be defined in the block registration. For example: 

```
const blockSettings = {
    // ... other settings

    tips: [
        'Quickly indent the text in a new list item by starting it with a space.',
    ]
}
```

If defined there, we may also consider placing these elsewhere in the UI: perhaps in the Block Sidebar (#16594) or the Block Placeholder (#16315). But the inserter is a good first step. 👍 

To start, we can pull from the list of block-specific tips listed in https://github.com/WordPress/gutenberg/issues/16595#issuecomment-518750188. In cases where there are multiple tips per block, we'll randomly chose one to display each time the block is hovered. 

**Example screenshot:** 

<img width="723" alt="Inserter-Paragraph" src="https://user-images.githubusercontent.com/1202812/63283795-2a223c80-c280-11e9-96e1-d99bb8dd973a.png">

**Comment by gracie** [comment]
We're working on getting specific ideas for [this issue here.](https://github.com/godaddy/coblocks/issues/817).

I'm excited about the initiative. In looking at the UI mock-up, I'm concerned the text will be ignore-able. It's small, largely black and white, and I didn't catch the lightbulb at the bottom until I looked for it.

I'm sorry for the critique without an alternative. Do you have ideas for how to make it stand out and catch a new user's eye to pay attention to it?

**Comment by richtabor** [comment]
This is a great idea @kjellr and something worth getting some action on. I'm looking forward to these!

**Comment by Mamaduka** [comment]
@richtabor, @jasmussen, should we close this issue? It's been inactive for some time now.

**Comment by jasmussen** [comment]
Agreed, let's close. It's not a bad idea, but it will also be easy to find and reopen if it becomes relevant again.
